src/todo.js

```diff
@@ -1,39 +1,32 @@
-export default class Todo {
-  static count = 0;
-
-  constructor(completed, description) {
-    // eslint-disable-next-line
-    this.index = this.constructor.count++;
-    this.completed = completed;
-    this.description = description;
-  }
+// eslint-disable-next-line
```

Comment:
- [OPTIONAL] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.